xml/System.Security.Cryptography/AesCcm.xml

```diff
@@ -86,13 +89,24 @@
         <Parameter Name="associatedData" Type="System.Byte[]" />
       </Parameters>
       <Docs>
-        <param name="nonce">To be added.</param>
-        <param name="ciphertext">To be added.</param>
-        <param name="tag">To be added.</param>
-        <param name="plaintext">To be added.</param>
-        <param name="associatedData">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="nonce">The nonce associated with this message, which must match the value provided during encryption.</param>
+        <param name="ciphertext">The encrypted content to decrypt.</param>
+        <param name="tag">The authentication tag produced for this message during encryption.</param>
+        <param name="plaintext">The byte array to receive the decrypted contents.</param>
+        <param name="associatedData">Extra data associated with this message, which must match the value provided during encryption.</param>
+        <summary>Decrypts the contents of <paramref name="ciphertext"/> into <paramref name="plaintext"/> if <paramref name="tag"/> can be validated.</summary>
```

Comment:
Just an observation FYI: it's best not to use parameter names in summary descriptions, since the summary appears both in the documentation for that member as well as in the member list. That makes the description in the member list unclear if the parameter name doesn't clearly describe the purpose of the parameter. That isn't the case with the AesCcm method descriptions, though.

Reply:
Would you make this then a bit more indirect, then?  Something like

```suggestion
        <summary>Decrypts the ciphertext into the provided destination buffer if the authentication tag can be validated.</summary>
```

Reply:
Yes, that's good, @bartonjs. Once you commit it, I'll merge.